gremlin-javascript/src/main/javascript/gremlin-javascript/test/integration/socket-connection-tests.js

```diff
@@ -49,7 +49,49 @@ describe('Connection', function () {
   });
 
   describe('#open()', function () {
+    it('should use the ws WebSocket when ws specific options are provided', function () {
+      let globalWebsocketCalls = 0;
+      globalThis.WebSocket = function () {
+        globalWebsocketCalls++;
+      };
+      const wsSpecificOptions = [
+        'headers',
+        'ca',
+        'cert',
+        'pfx',
+        'rejectUnauthorized',
+        'agent',
+        'perMessageDeflate',
+      ];
+      const allOptionTests = wsSpecificOptions.map((wsOption) => {
+        const connection = helper.getDriverRemoteConnection(`ws://localhost:${testServerPort}/401`, {
+          [wsOption]: 'this option is set',
+        });
+        return connection.open();
+      });
+      Promise.allSettled(allOptionTests).then(function () {
+        if (globalWebsocketCalls !== domUnsupportedOptions.length) {
```

Comment:
@KyleBoyer Where does `domUnsupportedOptions` come from?

And in this test, wouldn't `globalWebsocketCalls === 0`?

Reply:
Ahh, that should be `wsSpecificOptions`. I'll push that change shortly.

Correct, in this test the calls should be 0.

Reply:
@kmcginnes Changes pushed!